src/Microsoft.Tye.Hosting/ProxyService.cs

```diff
@@ -69,6 +69,15 @@ static int GetNextPort()
                                         binding.Port = GetNextPort();
                                     }
 
+                                    if (binding.Protocol == "http" || binding.Protocol == null)
+                                    {
+                                        binding.InternalPort = 80;
+                                    }
+                                    else if (binding.Protocol == "https")
+                                    {
+                                        binding.InternalPort = 443;
```

Comment:
Can you add internal port to the docs when you get a chance? I'm not 💯 on what it does, and so it's not there.

Reply:
It’s possible the same is bad as well. It’s really the container port.